local-tests/chainrunner/chain.py

```diff
@@ -62,7 +65,12 @@ def _set_flags(nodes, *args, **kwargs):
                 n.flags[k] = True
         for k, v in kwargs.items():
             for i, n in enumerate(nodes):
-                n.flags[k] = v + i if isinstance(v, Seq) else v
+                if isinstance(v, Seq):
+                    n.flags[k] = v + i
+                elif isinstance(v, list) and len(v) == len(nodes):
```

Comment:
should be added to documentation of the `set_flags...` functions

Reply:
Done